OpenTelemetry (S3M) Docs migration - merge March 28 (#1617)
SumoLogic/sumologic-documentation: docs/send-data/index.md

import useBaseUrl from '@docusaurus/useBaseUrl';

Learn how to send data to Sumo Logic, the starting point to using our observability and security solutions. In this section, we'll introduce the following concepts:

:::tip Designing your Deployment
* [How to choose a Collector and Source](/docs/send-data/choose-collector-source)
* [How to use the Collection UI](/docs/send-data/collection)
:::

<div className="box-wrapper" markdown="1">
<div className="box smallbox1 card">
  <div className="container">
  <a href="/docs/send-data/sumo-distribution-opentelemetry"><img src={useBaseUrl('img/send-data/otel-color.svg')} alt="icon" width="40"/><h4>Sumo Logic Distro for OpenTelemetry</h4></a>
  <p>Our OTel distro is a single unified agent to send Logs, Metrics, Traces, and more.</p>
  </div>
</div>
<div className="box smallbox2 card">
  <div className="container">
  <a href="/docs/send-data/installed-collectors"><img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>Installed Collectors</h4></a>

>   <p>Install a collector in your environment to receive logs and metrics from your Sources.</p>

Reviewer: ```suggestion
  <p>Installed collector is our proprietary agent that you can deploy in your environment to collect logs and some metrics sources.</p>
```

Propose changing the text slightly here, to make sure we higlight that this is also an agent.

  </div>
</div>
<div className="box smallbox3 card">
  <div className="container">

>   <a href="/docs/send-data/hosted-collectors/configure-hosted-collector"><img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>Hosted Collectors</h4></a>

Reviewer: ```suggestion
  <a href="/docs/send-data/hosted-collectors/"><img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>Hosted Collectors</h4></a>
```

Author: good catch!

>   <p>Learn how to configure a Sumo-hosted collector and add Sources from a variety of services.</p>

Reviewer: ```suggestion
  <p>Hosted Collector allows you to send data to Sumo Logic without deploying an agent. Used for Cloud to Cloud collection from AWS, Azure, GCP and various SaaS tools. It can also be used to send data directly to Sumo endpoint using your custom collection method</p>
```

Author: Can you shrink this down to < 125 characters? This is great though, we can definitely use this in the actual Hosted Collectors doc!

  <p>Collect logs and metrics from a variety of other data sources.</p>
  </div>
</div>
<div className="box smallbox5 card">
  <div className="container">
  <a href="/docs/send-data/use-json-configure-sources/json-parameters-hosted-sources"><img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>JSON Configuration</h4></a>
  <p>Learn how to configure Installed Collector and Hosted Collector sources using JSON files.</p>
  </div>
</div>
  <div className="box smallbox6 card">
    <div className="container">
    <a href="/docs/send-data/reference-information/time-reference"><img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>Timestamps, Time Zones Reference</h4></a>
    <p>Learn about supported timestamps, date formatting, and more. </p>
    </div>
  </div>
  <div className="box smallbox7 card">
    <div className="container">
    <a href="/docs/send-data/reference-information/use-wildcards-paths"><img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>Wildcards in Paths</h4></a>

>     <p>Use wildcards in Source paths to collect all files of a certain type.</p>

Reviewer: Why do we have this section in the Send Data Page ? Shouldn't this be within the Installed Collector, Otel Collector Pages, since it relates how we should set things up?

Author: So this does not apply to Hosted Collectors? Installed and OTel only?

Author: In the current structure, it looks like all reference material is here: https://help.sumologic.com/docs/send-data/reference-information. We can definitely look into this after S3M.
